react/apollo/index.js

```diff
@@ -50,6 +56,14 @@ export const initApolloClient = ({ token: X_AUTH_TOKEN, currentRoute, isLoggedIn
         __typename: 'LoginStatus',
         isLoggedIn,
       },
+      cookies: {
+        __typename: 'Cookies',
+      },
+    },
+    resolvers: {
+      Cookies: {
+        get: (_obj, args) => cookies[args.name] || null,
+      },
```

Comment:
I do think we should update this using mutations but I don't feel like dealing with it right now.